src/pages/index.tsx

```diff
@@ -52,6 +52,10 @@ const DAppExample: NextPage = () => {
             }
           }
         });
+      } else {
+        alert(
+          'No extension installed. Please install XDEFI Wallet at https://chromewebstore.google.com/detail/xdefi-wallet/hmeobnfnfcmdkdcmlblgagmfpfboieaf'
```

Comment:
Can we change to display (in the blank page) and not an `alert` --  just this and a link to https://xdefi.io @HoangVD2

Reply:
updated @davidp94